packages/ui-extensions-react/src/use-template-variable.ts

```diff
@@ -70,29 +70,6 @@ function useTemplateVariable(
         setVariable(value);
     }, [context?.dashboard?.templateVars, variableName]);
 
-    React.useEffect(() => {
-        const unsubscribe = client.events.on(
-            uiExtensionsSDK.EventType.DASHBOARD_TEMPLATE_VAR_CHANGE,
-            (
-                templateVariables: uiExtensionsSDK.TemplateVariableValue[]
-            ): void => {
-                const value = findTemplateVariableValue(
-                    variableName,
-                    templateVariables
-                );
-                if (value == null) {
-                    return;
-                }
-
-                setVariable(value);
-            }
-        );
-
-        return () => {
-            unsubscribe();
-        };
-    }, [client.events, variableName]);
-
```

Comment:
@joneshf-dd I believe we are safe to _only_ use the context change event here, having verified that it always fires when template vars change

Reply:
Yeah, that seems right! Thanks for grabbing this!